src/components/rangeslider/attributes.js

```diff
@@ -33,10 +33,6 @@ module.exports = {
         role: 'style',
         description: 'Sets the background color of the range slider.'
     },
-    initialrange: {
-        valType: 'data_array',
-        description: 'The starting range of the range slider, in data coordinates.'
```

Comment:
Any plans for re-adding this in this PR?

For the record, I'm ok with not having it in the first iteration.